server/WebAPI/Controllers/UserController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Services;
+
+namespace WebAPI.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")]
+    public class UserController : Controller
```

Comment:
are you sure that we don't have controller and repository for user? we have a lot of functionality with user